haystack/preview/components/samplers/top_p.py

```diff
@@ -7,7 +7,7 @@
 logger = logging.getLogger(__name__)
 
 
-with LazyImport(message="Run 'pip install torch>=1.13'") as torch_import:
+with LazyImport(message="Run 'pip install \"torch>=1.13\"'") as torch_import:
```

Comment:
@vblagoje why are we pinning `torch>=1.13`?

Reply:
Check transformers project file. Could've been simply copied from there